_data/implementations.yml

```
@@ -137,6 +137,16 @@
     url: https://github.com/18F
   notes: "This [Ruby on Rails](http://rubyonrails.org/) gem is not currently maintained."
 
+- name: Comet
+  distribution: React
+  id: comet-uswds
+  url: https://github.com/MetroStar/comet
+  author:
+    name: MetroStar
+    url: https://github.com/MetroStar
+  version: 3.5.0
+  notes: "React with TypeScript implementation of USWDS, providing additional modular support for custom components, data visualization, and command line tools. Sample starter app included that is focused on accelerating and scaling design across the enterprise."
```

Added a couple of suggestions to help improve clarity. 

```suggestion
  notes: "A React with TypeScript component library based on USWDS 3.0. This library provides modular support for custom components, data visualization, and command line tools. Comet also includes a sample starter app that is focused on accelerating and scaling design across the enterprise."
```

---

Updated as requested. Thank you!